### ISSUE TITLE
cookieMonster error..

### ISSUE THREAD

**Issue by RuiS2** [IgrutinovicL/TrovoViewbot#11]
hey bro! 
so while running cookieMonster i got this error at the beginning 
![1](https://user-images.githubusercontent.com/102841860/161363174-0f4b405a-bd5f-49b5-a8a5-e543a2db1ed1.png)

and when i added the line ( if __name__ == '__main__': ) 
![3](https://user-images.githubusercontent.com/102841860/161363349-bdee138e-9468-4672-b93a-c3b375497196.png)

i got this error ! .. 
![2](https://user-images.githubusercontent.com/102841860/161363175-89cf3a42-90f9-4b6d-b3d4-f5d8700e9c39.png)
tried some solutions but nothing seems to work .. 

any idea on whats going on here ?!

**Comment by IgrutinovicL** [comment]
First of all never put `if __name__ == "__main__"` on top of a file. Always put it above the main function.
The OS is not defined error is because of the previous thing i said.

So move it down before main().

Use tabs not spaces when moving codes into a block.

You can remove the `os.system("cls")` because that only clears the console.

**Comment by RuiS2** [comment]
thank you so much .. this worked out 
another thing i wanted to ask about .. am getting errors like " [!] Could not inject cookies " " [!] Error when opening URL. " 
any idea ?

thanks ^^

**Comment by IgrutinovicL** [comment]
Try [Proxyscrape](https://proxyscrape.com/home) for some good proxies.

**Comment by RuiS2** [comment]
what about the injecting prob ?

**Comment by IgrutinovicL** [comment]
Oh sorry, about the injection. Did you make cookies with monster? Also did the website load when the viewbot tried to inject?

**Comment by RuiS2** [comment]
yes i did make the cookies using monster, chrome opens up i go to trovo and log into an account then i name the package and thats it right ? .. 

the website does not load .. keeps on closing and opening new windows + am just trying 2 bots 
![3 (2)](https://user-images.githubusercontent.com/102841860/161397290-4d17fb47-4b30-4c94-a56e-9dc62316005f.png)
![5](https://user-images.githubusercontent.com/102841860/161397292-5b4fb141-92f0-4c11-ad9e-574af1fb24a8.png)

**Comment by IgrutinovicL** [comment]
Hmmm, I think it has something to do with the GPU because of the D3D11 device was removed. Try to add this line:

`options.add_argument('--disable-gpu')`

After the last `options.add_argument`

I have never seen this error before. I will try my best to help you.

**Comment by RuiS2** [comment]
that might be the case .. am using win 7 on some old laptop for sake of learning.
i will try that line out .. thank you so much bro, i really appreciate your concern ^^

**Comment by IgrutinovicL** [comment]
Okay, I won't close this issue so you can update. 😄

**Comment by RuiS2** [comment]
not really sure whats going on .. keeps on loading pages trying to reach trovo but nothing seems to happen

![6](https://user-images.githubusercontent.com/102841860/161397893-05d00446-c92e-4da6-acb7-5a4f4b22532b.png)

![7](https://user-images.githubusercontent.com/102841860/161397899-e3cfc1e9-ffa6-462c-aa5d-49e364165c5e.png)

![8](https://user-images.githubusercontent.com/102841860/161397901-589a7f48-5ab7-46e5-aea9-122e3b79693d.png)

**Comment by IgrutinovicL** [comment]
Are you sure that the proxies are working? 

Try proxies from [Proxy scrape](https://proxyscrape.com/free-proxy-list)

Plus where it says that it's unable to locate the element, its saying that the element has probably changed it's location on the page. You can try to find the vcp control panel yourself or you can wait for me to find it. But I will not be working on the bot for a while since I am not home.

**Comment by RuiS2** [comment]
am already using proxies from proxy scrape .. 
i will try my best, thank you again and gd luck ^^

**Comment by IgrutinovicL** [comment]
Here is some documentation on how to find elements with python/selenium:

[Docs](https://selenium-python.readthedocs.io/locating-elements.html)

And a little help with inspect element:

[Inspect element](https://www.toolsqa.com/selenium-webdriver/inspect-element-in-chrome/)

**Comment by RuiS2** [comment]
great!

**Comment by guiinho** [comment]
solution
![Untitled](https://user-images.githubusercontent.com/102909221/161434043-a099fff2-9b0b-4295-b0d9-ce8673f3508c.png)
?

**Comment by IgrutinovicL** [comment]
Does it open the trovo.live page?

**Comment by guiinho** [comment]
this error persists what can I do?
![image](https://user-images.githubusercontent.com/102909221/161628227-65c0c64c-e5e1-4a43-b954-4a02109d5d9a.png)

**Comment by IgrutinovicL** [comment]
I legit have never seen this error. I really want to help everybody but I just don't know what this is. Can you tell me all the steps you have taken to make this happen?

**Comment by IgrutinovicL** [comment]
So maybe i can recreate this issue on my PC and find a solution

**Comment by guiinho** [comment]
I simply took the cookies with CookieMonster i opened the bot.py and did the process of configuring and started to put to certain errors  sorry for to disturb man

**Comment by guiinho** [comment]
this bot counts assisted hours ?

**Comment by IgrutinovicL** [comment]
It counts hours.

**Comment by guiinho** [comment]
proxies are slow so maybe that's why you have this error ''Could not inject cookies''
